tests/operator_test.go

```diff
@@ -402,6 +510,173 @@ var _ = Describe("Operator", func() {
 		waitForKv(originalKv)
 		allPodsAreReady(tests.KubeVirtVersionTag)
 
+		if workDir != "" {
+			err = os.RemoveAll(workDir)
+			Expect(err).ToNot(HaveOccurred())
+			workDir = ""
```

Comment:
This pattern comes from elsewhere in the codebase, but ... what happens if "RemoveAll" actually generates an error here, doesn't "workDir" end up non-empty? perhaps swap lines 515 and 516?

Reply:
done